README.rst

```diff
@@ -97,29 +98,29 @@ Nothing special.
 Executing SCons Without Installing
 ==================================
 
-You can execute the local SCons directly from the src/ subdirectory by first
-setting the SCONS_LIB_DIR environment variable to the local src/engine
-subdirectory, and then executing the local src/script/scons.py script to
+You can execute the local SCons directly from the SCons subdirectory by first
+setting the SCONS_LIB_DIR environment variable to the local SCons
+subdirectory, and then executing the local scripts/scons.py script to
 populate the build/scons/ subdirectory.  You would do this as follows on a
 Linux or UNIX system (using sh or a derivative like bash or ksh)::
 
         $ setenv MYSCONS=`pwd`
-        $ python $MYSCONS/script/scons.py [arguments]
+        $ python $MYSCONS/scripts/scons.py [arguments]
 
 Or on Windows::
 
-        C:\scons>set MYSCONS=%cd%\src
-        C:\scons>python %MYSCONS%\script\scons.py [arguments]
+        C:\scons>set MYSCONS=%cd%
+        C:\scons>python %MYSCONS%\scripts\scons.py [arguments]
 
 An alternative approach is to skip the above and use::
 
-        $ python script/scons.py [arguments]
+        $ python scripts/scons.py [arguments]
```

Comment:
I think this should be the primary suggestion, and the others be alternative?

Reply:
it's what I do all the time, but I didn't want to apply my own biases (one more time)... that's why I suggested some other review of the README might be valuable. (also, I didn't touch the "sourceforge version", I presume that's okay?)

Reply:
They should have basically the same content. I think the issue was sourceforge wasn't rendering the buttons from appveyor/etc so they had to be removed.

Reply:
Funky. Okay, I can make them the same except the stuff at the top.